test/src/utils/testUtils.js

```diff
@@ -206,6 +206,28 @@ describe('test Utils', function() {
     console = oldConsole;
   });
 
+  it('test getModuleKey', function() {
+    const srcFolderPath = path.join(__dirname, '..');
```

Comment:
can you maybe use one of the sample tests folder which only has a few tests?

Reply:
you can use any of the sample test folders in the `test` directory which has a few sub folders.

Reply:
> you can use any of the sample test folders in the `test` directory which has a few sub folders.

Done, I used the current folder the test lies in, I hope this will get the PR merged, it's been like a year since we started having this confusing logs.